core/src/main/scala/kafka/javaapi/FetchResponse.scala

```diff
@@ -28,7 +28,7 @@ class FetchResponse(private val underlying: kafka.api.FetchResponse) {
 
   def hasError = underlying.hasError
 
-  def errorCode(topic: String, partition: Int) = underlying.errorCode(topic, partition)
+  def error(topic: String, partition: Int) = underlying.error(topic, partition)
```

Comment:
Maybe we should keep `errorCode` for compatibility. My understanding is that the request classes under `javaapi` are public. There are a couple more of these below.